packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/DocumentController.java

```diff
@@ -191,48 +193,48 @@ public ResponseEntity<Integer> uploadDocumentFromGithub(
 		log.debug("Uploading Document file from github to dataset {}", documentId);
 
 		//download file from GitHub
-		String fileString = gitHubProxy.getGithubCode(repoOwnerAndName, path).getBody();
-		HttpEntity fileEntity = new StringEntity(fileString, ContentType.TEXT_PLAIN);
+		final String fileString = gitHubProxy.getGithubCode(repoOwnerAndName, path).getBody();
+		final HttpEntity fileEntity = new StringEntity(fileString, ContentType.TEXT_PLAIN);
 		return uploadDocumentHelper(documentId, filename, fileEntity);
 
 	}
 
 	@PostMapping(value = "/createDocumentFromXDD")
 	public ResponseEntity<AddDocumentAssetFromXDDResponse> createDocumentFromXDD(
-		@RequestBody AddDocumentAssetFromXDDRequest body
+		@RequestBody final AddDocumentAssetFromXDDRequest body
```

Comment:
If this is all just formatting can you revert everything in here (`createDocumentFromXDD`)? Its going to cause a LOT of grief with my PR for xdd extractions to document assets...